dashboard/src/t5gweb/t5gweb.py

```diff
@@ -136,6 +137,80 @@ def get_new_comments():
 
     return accounts
 
+def get_trending_cards():
+        # Set the default configuration values
+    cfg = set_cfg()
+    cfg['query'] = "case_summary:*webscale* OR case_tags:*shift_telco5g* OR case_summary:*cnv,* OR case_tags:*cnv*"    
+    try:
+        conn = jira.JIRA(cfg['options'], basic_auth=(cfg['jira_user'], cfg['jira_pass']))
+    except jira.exceptions as e:
+        if e.status_code == 401:
+            print("Login to JIRA failed. Check your username and password")
+            exit (1)
+    board = libtelco5g.get_board_id(conn, cfg['board'])
+    sprint = libtelco5g.get_latest_sprint(conn, board.id, cfg['sprintname'])
+    cards = conn.search_issues('component = "KNI Labs & Field" AND (project = KNIECO OR project = KNIP AND issuetype = Epic AND status != Obsolete) AND labels = "Trends" AND updated >= "2021-07-01" AND updated <= "2021-09-30" ORDER BY Rank ASC', maxResults=1000)
```

Comment:
every quarter we're gonna have to update this bit. can you add some logic to detect the last quarter rather than hard code it?